public/apps/account/test/account-nav-button.test.tsx

```diff
@@ -56,6 +56,7 @@ describe('Account navigation button', () => {
         username={userName}
         tenant="tenant1"
         config={config as any}
+        currAuthType={'dummy'}
```

Comment:
same question for the 3 changes in this file: https://github.com/opensearch-project/security-dashboards-plugin/pull/1110/files#r1006454249

Reply:
The currentAuthType is not being used until logout button triggers onClick function. So the return value for fetchCurrentAuthType is just a placeholder here.